lib/api/collection_config.rb

```diff
@@ -53,12 +53,12 @@ def typed_collection_actions(collection_name, target)
       self[collection_name]["#{target}_actions".to_sym]
     end
 
-    def typed_subcollection_actions(collection_name, subcollection_name)
-      self[collection_name]["#{subcollection_name}_subcollection_actions".to_sym]
+    def typed_subcollection_actions(collection_name, subcollection_name, target)
```

Comment:
A lot of changes because of the required parameter,  couldn't you make target optional and keep the signature backward compatible ?